mono/utils/mono-dl-wasm.c

```diff
@@ -0,0 +1,75 @@
+#include <config.h>
+
+#if defined (HOST_WASM)
+
+#include "mono/utils/mono-dl.h"
+#include "mono/utils/mono-embed.h"
+#include "mono/utils/mono-path.h"
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <ctype.h>
+#include <string.h>
+#include <glib.h>
+
+const char *
+mono_dl_get_so_prefix (void)
+{
+	return "";
+}
+const char **
+mono_dl_get_so_suffixes (void)
+{
+	static const char *suffixes[] = {
+		".wasm",
+		"",
+	};
+	return suffixes;
+}
+
+int
+mono_dl_get_executable_path (char *buf, int buflen)
+{
+	strncpy (buf, "/managed", buflen);
```

Comment:
Yeah, probably. Good thing it's not particularly useful, given dynamic loading is not supported at this stage of the port.